dialogplus/src/main/java/com/orhanobut/dialogplus/ResolverDrawerLayout.java

```diff
@@ -0,0 +1,814 @@
+package com.orhanobut.dialogplus;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Rect;
+import android.os.Bundle;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.VelocityTracker;
+import android.view.View;
+import android.view.ViewConfiguration;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.view.ViewTreeObserver;
+import android.view.accessibility.AccessibilityEvent;
+import android.view.accessibility.AccessibilityNodeInfo;
+import android.view.animation.AnimationUtils;
+import android.widget.AbsListView;
+import android.widget.OverScroller;
+
+public class ResolverDrawerLayout extends ViewGroup {
+    private static final String TAG = "ResolverDrawerLayout";
+
+    /**
+     * Max width of the whole drawer layout
+     */
+    private int mMaxWidth;
```

Comment:
Convention for library is not using m prefix for variable names. i.e. writing maxWidth instead of mMaxWidth makes sense. All fields should be renamed in this class then